hash_practice/exercises.py

```diff
@@ -1,17 +1,61 @@
-
+# Time Complexity: ?
+# Space Complexity: O(n)
 def grouped_anagrams(strings):

```

Comment:
👍 If the word is an English word (so limited in length) the time complexity is O(n)